deploy/openj9-chart/README.md

```diff
@@ -0,0 +1,106 @@
+# OpenJ9 Helm Chart
+
+## Introduction
+
+Eclipse OpenJ9 is an independent implementation of a Java Virtual Machine.
+
+The OpenJ9 Chart allows you to deploy and manage java applications running on OpenJ9 into OKD or OpenShift clusters. You can also perform Day-2 operations such as rolling update your application, enabling JITServer technology for continued performance enhancements or gathering JVM dumps using the operator.
```

Comment:
Also, here we specify "OKD or OpenShift clusters". Don't we support any Kubernetes distribution?